vcd_cli/user.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import division
+
+import click
+from pyvcloud.vcd.org import Org
+
+from vcd_cli.utils import stderr, restore_session, stdout
+from vcd_cli.vcd import vcd
+
+
+@vcd.group(short_help='work with users in current organization')
+@click.pass_context
+def user(ctx):
+    """Work with users in current organization.
+
+\b
+    Examples
+        vcd user create 'user_name' 'password' 'role_name'
```

Comment:
single quotes can be removed when there is no blank space or special character in the token

Reply:
Fixed.